src/server.ts

```diff
@@ -1,18 +1,28 @@
-import { Express } from 'express';
-import * as express from 'express';
+// Load enviornment
+require('dotenv').config();
+import * as e from 'express';
 import * as bodyParser from 'body-parser';
 import * as cors from 'cors';
 import { headers } from './middleware/headers';
 import { routes } from './routes';
+
 import { createRedisClient } from './clients/redis/redis.client';
+import { db } from './clients/db/mongoose.client';
+import { initSlackBot } from './bots/slack/init';
 
 // Creates Redis Client
 export const redis = createRedisClient();
 
-// Server Setup
-const port = process.env.SERVER_PORT || 4000;
+// Connect to DB via Mongoose
+db();
+
+// Initialize Slackbot
+initSlackBot();
+
+// Express Server Setup
+const port = process.env.EXPRESS_SERVER_PORT || 4000;
```

Comment:
FYI, the PORT env is used by Botkit to edit what port we want it to use. We can't run both servers on this port.

Reply:
botkit is running on 3000 by default.  I think it is just a wrapper for express